tests/scale/test_load.py

```diff
@@ -51,7 +51,7 @@
 log = logging.getLogger(__name__)
 
 SHARED_ROLE = "jenkins-role"
-DOCKER_IMAGE="benclarkwood/dind:3"
+DOCKER_IMAGE="mesosphere/jenkins-dind:scale"
```

Comment:
It would be ideal to have something unique (SHA may be?!) in the tag.

Reply:
we dont have the docker file for this image since it was bens